test/unit/gateways/ixopay_test.rb

```diff
@@ -4,7 +4,12 @@ class IxopayTest < Test::Unit::TestCase
   include CommStub
 
   def setup
-    @gateway = IxopayGateway.new(username: 'username', password: 'password', secret: 'secret')
+    @gateway = IxopayGateway.new(
+      username: 'username',
+      password: 'password',
+      secret:   'secret',
+      api_key:  'api_key'
```

Comment:
Just added the newly-required API key here.

Reply:
Now that this has been updated, the error message included in `test_failed_authentication` below should also be updated.

Reply:
Done.